x/oracle/keeper/keeper_test.go

```diff
@@ -0,0 +1,60 @@
+package keeper_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/99designs/keyring"
```

Comment:
It seems like we want to avoid this reference and instead rely on the Cosmos SDK replacement ... is it possible to create this test without this?

Reply:
Sure, I can replace it with the Cosmos SDK one.